tests/stdlib/tregistry.nim

```diff
@@ -0,0 +1,9 @@
+discard """
+  disabled: "unix"
```

Comment:
Just a comment on testament rather than this PR - is there no functionality to _only_ enable test for a specific target? If there's not, then I guess that it might be a good idea to add an optional like that.

Reply:
make sense

Reply:
until then this works:
```nim
when defined(unix):
  xxx
```
more generally, I'd rather have most of the logic in pure nim rather than in testament's spec language

Reply:
It still waste some resources?